docs/notebooks/word2vec.ipynb

```diff
@@ -1295,6 +1295,31 @@
     "print(train_times_table)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Adding Word2Vec \"model to dict\" method to production pipeline\n",
+    "Suppose, we still want more performance improvement in production. \n",
+    "One good way is to cache all the similar words in a dictionary.\n",
+    "\n",
```

Comment:
No needed `\n` after each sentence + you should add a link to the concrete issue in the notebook.